spec/ruby/library/win32ole/fixtures/classes.rb

```diff
@@ -1,4 +1,8 @@
+require 'win32ole'
```

Comment:
I'm afraid that this would be fail in some CI environments except Windows.

Reply:
It passed in the builds that completed test-all.  All the files I modified required it after verifying that it was windows, so it was never required in a non windows environment...

Reply:
Also, I used MSXML because it has a few events, so it could be a generic OLE object that would work in most tests.  Appveyor testing seems to show that their test systems' resources can vary quite a bit, so a lightweight object seemed beneficial, especially when running testing parallel...

Reply:
Thank you. I misunderstood.

Reply:
No problem.  I've worked quite a bit with test-all & test-spec in terms of reliability.  I know the norm is to have all requires at the top of a file, but require == drive access, so I'll often test before requiring on platform specific tests...

Reply:
Sorry, but unfortunately it failed on some environment.

https://github.com/ruby/ruby/commit/091c04df537425d9e73b8b2f079522a05df93119

Reply:
I replied in https://bugs.ruby-lang.org/issues/15239?next_issue_id=14867.

The patch cannot be broken into separate commits, as detailed in the issue.  I think the PR and the patch in the Ruby issue both conflict with the repo as it is now.  I'm not a committer...